.pre-commit-config.yaml

```
@@ -29,4 +29,4 @@ repos:
   hooks:
   - id: mypy
     files: src
-    additional_dependencies: [attrs==19.3.0]
+    additional_dependencies: [attrs==19.3.0, types-Deprecated]
```

```suggestion
    additional_dependencies: [attrs==19.3.0, types-deprecated]
```

Ultra minor, but packages are usually lower case, even though mypy suggests the cased version. I sometimes see "Cython" instead of "cython", but usually it's lower case everywhere.

---

Fair enough. I got this example online, actually.